protocols/netty/src/main/java/org/apache/james/protocols/netty/NettyProtocolTransport.java

```diff
@@ -126,26 +125,8 @@ protected void writeToClient(InputStream in, ProtocolSession session, boolean st
                     channel.write(new ChunkedStream(new ExceptionInputStream(e)));
                 }
                 return;

```

Comment:
what happens else?

Reply:
see https://github.com/linagora/james-project/pull/3302/files/8b3db4a16b2b8c94fcb9c014d1a45099ae8d123d..f025913061013cd51172b1528aed794de326a0a2#diff-7a9220d5581cf0c1ae4275520c3091d0L149 it writes it another way